online/onlmonserver/OnlMonCham.cc

```diff
@@ -121,20 +121,20 @@ int OnlMonCham::DrawMonitor()
 {
   OnlMonCanvas* can0 = GetCanvas(0);
   TPad* pad0 = can0->GetMainPad();
-  pad0->SetGrid();
```

Comment:
Here is an extra change that I made in this PR.  The axis grid of all OnlMon plots is (correctly) drawn now.  You can look at the OnlMon HTML output of run 2868, for example, to see how they appear.